Dockerfile

```diff
@@ -25,6 +25,7 @@ ENV HOME /home/${NEW_USER}
 ENV PYENV_ROOT=/home/${NEW_USER}/.pyenv
 ENV PATH=$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
 ENV JAVA_HOME=/opt/jdk-10
+ENV JAVA10_HOME=/opt/jdk-10
```

Comment:
This seems unrelated?

Reply:
Yes this came from this commit https://github.com/elastic/rally/pull/538/commits/9ddb3933dbb49853ea77540fa3c198d02bdf0cbe to fix the it tests in Docker. I'll pull it out and submit separately.